### PR TITLE
fix shadow dom test

### PR DESCRIPTION
**Pull request by notwaldorf**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
     "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "*",
+    "web-component-tester": "polymer/web-component-tester#3.4.0",
```
Comment (cdata) [inline bower.json]: :boar:

File: test/basic.html [hunk 2]
```diff
@@ -16,9 +16,6 @@
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
-  <script src="../../test-fixture/test-fixture-mocha.js"></script>
-
-  <link rel="import" href="../../test-fixture/test-fixture.html">
```
Comment (cdata) [inline test/basic.html]: :bear:

File: test/basic.html [hunk 3]
```diff
@@ -58,7 +55,7 @@
       var f, img;
       setup(function () {
         f = fixture('basic');
-        img = f.querySelector('iron-image');
+        img = f.$$('iron-image');
```
Comment (cdata) [inline test/basic.html]: `$$` = scoped query selector?

**Comment by cdata** [comment]
Small question, otherwise LGTM :+1:

File: test/basic.html (hunk 3 above)
Comment (notwaldorf) [inline test/basic.html]: yeah! https://www.polymer-project.org/1.0/docs/devguide/local-dom.html#work-with-local-dom (the last thing in that section).

Is it weird? `$$` goes through the shadow boundary, which this test needs

**Pull request merged**